modules/ROOT/pages/understanding-orchestration-using-mule.adoc

```diff
@@ -0,0 +1,76 @@
+= Understanding Orchestration Using Mule
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, studio, mule esb, orchestration
+
+Orchestration is how Mule facilitates the process of moving data through a Mule implementation based on flows. Orchestration can be seen as combining service calls to create higher-level, more useful composite services, and implies implementing business-level processes combining business-specific services across applications and information systems.
+
+== Orchestration and Flows
+
+Orchestration involves decoupling point-to-point integration in favor of chains of operations that can be reused or changed as systems or business needs change. It involves moving beyond patterns into a service-based framework, where you wrap multiple components with sources to decouple them from protocols and routing requirements. This enables multiple services to work as one when implementation requirements demand it. Some common use cases that flows easily orchestrate include:
+
+* Simple integration
+* Scheduling data processing
+* Connecting cloud and on-premise applications
+* Event processing, where multiple services must be composed and must operate simultaneously
+
+== How Flows Enable the Implementation of Orchestration
+
+Flows enable new ways to orchestrate service mediation. For example, you can combine an arbitrary number of sources, routers, and components that work together to achieve implementation requirements. Flows enable you to orchestrate new forms of data movement based on flow-specific routers. Orchestration enables you to create and reuse flows and component chains, chain together multiple components in one flow, and integrate sources or connectors anywhere in a flow.
+
+== When to Use Orchestration
+
+Use orchestration when you need to combine existing services to meet business requirements. For example, you might use orchestration when you're adding a new employee who needs to be entered in both the payroll system and HR system. You might also use orchestration for order processing. In this scenario, you can use orchestration to accept an order, make sure the order is in the inventory system., use a credit check system to ensure that you trust the customer's credit, and enter the order into billing and shipping systems.
```

Comment:
There is an extra dot in this line ..._inventory system., use a credit_....

Reply:
OK